### ISSUE TITLE
Update online documentation for 2.0.0 release

### ISSUE THREAD

**Issue by cmarchand** [xspec/xspec-maven-plugin-1#43]
Many parameters change in 2.0.0 release, and online documentation must be updated.
Just a reminder.

**Comment by thendarion** [comment]
yeah, I'm kinda reluctant to upgrade until I know what's new and what can potentially break :D
